### PR TITLE
Version 1.0 changes

### PR DESCRIPTION
**Pull request by dturner**

### CODE REVIEW

File: include/oboe/AudioStream.h [hunk 1]
```diff
@@ -268,7 +268,6 @@ class AudioStream : public AudioStreamBase {
 
     /**
      * Get the estimated time that the frame at `framePosition` entered or left the audio processing
```
Comment (philburk) [inline include/oboe/AudioStream.h]: incomplete sentence

File: include/oboe/AudioStream.h [hunk 2]
```diff
@@ -280,14 +279,31 @@ class AudioStream : public AudioStreamBase {
...
-     * (if the operation is successful)
      */
     virtual Result getTimestamp(clockid_t clockId,
                                 int64_t *framePosition,
                                 int64_t *timeNanoseconds) {
         return Result::ErrorUnimplemented;
     }
 
+    /**
+     * Get the estimated time that the frame at `framePosition` entered or left the audio processing
```
Comment (philburk) [inline include/oboe/AudioStream.h]: incomplete sentence

File: include/oboe/Definitions.h [hunk 3]
```diff
@@ -469,6 +469,13 @@ namespace oboe {
 
     };
 
+    /**
+     * The time at which the frame at `position` was presented
+     */
+    struct FrameTimestamp {
+        int64_t position;
+        int64_t timestamp;
```
Comment (philburk) [inline include/oboe/Definitions.h]: position // in frames
timestamp // in nanoseconds

File: include/oboe/AudioStream.h (hunk 1 above)
Comment (dturner) [reply include/oboe/AudioStream.h]: Fixed

File: include/oboe/AudioStream.h (hunk 2 above)
Comment (dturner) [reply include/oboe/AudioStream.h]: fixed

File: include/oboe/Definitions.h (hunk 3 above)
Comment (dturner) [reply include/oboe/Definitions.h]: fixed

**Comment by philburk** [comment]
Do you want to merge this?

**Comment by philburk** [comment]
@dturner - do you want merge these?

**Comment by dturner** [comment]
I'd prefer to wait until the launch video is ready. Just need one more approval.

**Pull request merged**